react-hot-button/my-app/src/Button.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import './Button.css';
+
+export function Button() {
+  const [count, setCount] = useState(0);
+  const [color, setColor] = useState('indigo');
```

Comment:
Since the color can be computed from the count, it's best to not put `color` in state. Usually, if a value can be computed from another state or a prop, you just compute it directly without state.

So, your handleClick would simply increment the count.

Then, in the main flow of your rendering code, you declare a variable named `color` and compute its value. Then use it to set the className.